requirements.txt

```diff
@@ -20,7 +20,7 @@ git+git://github.com/sensu/sensu-plugin-python@bf00ac33
 # so we pull from git
 git+git://github.com/thefactory/marathon-python@c1e0c341#egg=marathon
 # While task_processing is in rapid development mode, we install it from git
-git+git://github.com/Yelp/task_processing@95cc2f8b#egg=task_processing[mesos_executor]
+git+git://github.com/Yelp/task_processing#egg=task_processing[mesos_executor]
```

Comment:
i'm hesitant to dropping specific git sha. shipping features unrelated to taskproc will always include latest changes on taskproc - that will trip up people.